src/routes/docs/configuring/git-integration/+page.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Manage repositories with Git
+---
+
+Integration with Git allows you to work more easily with your projects. You can create and manage branches directly from Files. You can pull and push changes to remote repos, and open them in VS Code.```
+
+## Using Git in Files
+
+No additional setup is required, opening a Git folder in Files will automatically detect and display the branch name on the Status Bar.
+
+### Switching branches
+
+Clicking the branch name will open a flyout with the list of available branches, you'll also be able to create a new branch from this flyout.
+## How to see repository branches
+
+To see repository branches, follow these steps:
+
+- Navigate to the folder containing the repository.
+- Click the name of the current branch in the bottom right corner.
+
+You will see a list of all local and remote repository branches.
+
+## How to create a new branch
+
+To create a new branch, follow these steps:
+
+- Navigate to the folder containing the repository.
+- Click the name of the current branch in the bottom right corner.
+- Click the "Create branch" button in the menu that appears.
+- Enter the name of the new branch, select the base of the new branch and choose whether you want to switch to the new branch after creation.
+- Click the "Create" button.
+
+## How to switch to another branch
+
+To switch to another branch, follow these steps:
+
+- Navigate to the folder containing the repository.
+- Click the name of the current branch in the bottom right corner.
+- Click the branch you want to switch to in the menu that appears.
+
+## How to pull changes from a remote repository
```

Comment:
Can you update this to reflect the changes in the preview builds?

Reply:
Sure!